packages/datatrak-web/src/features/Survey/SurveyScreen.tsx

```diff
@@ -205,7 +211,7 @@ export const SurveyScreen = () => {
                   Back
                 </Button>
                 <ButtonGroup>
-                  <Button variant="outlined" to={ROUTES.SURVEY_SELECT}>
+                  <Button variant="outlined" onClick={openCancelModal}>
```

Comment:
This is extremely minor but the color is slightly off for the outlined button. It has some opacity which it shouldn't have. Probably best to just add a style in the button component for when mui-outlined is set